cfme/tests/containers/test_containers_default_project_replicators.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import CheckboxTable
+from cfme.containers.replicator import Replicator
+from itertools import product
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        (lambda: current_version() < "5.6") or (lambda provider: provider.version < 3.2)),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+
+
+REPLICATORS_PROPERTIES_FIELDS = ['Name', 'Creation timestamp', 'Resource version',
```

Comment:
The thing is, the product returns list of n-tuples but there is only one argument, therefore tuple gets passed in the test and the ugly id is generated. Consider splitting the product in its respective values by addign multiple arguments, then it will work as you would expected :)